src/Features/CSharp/Portable/Completion/KeywordRecommenders/StaticKeywordRecommender.cs

```diff
@@ -81,7 +81,7 @@ protected override bool IsValidContext(int position, CSharpSyntaxContext context
         {
             return
                 context.IsGlobalStatementContext ||
-                context.TargetToken.IsUsingKeywordInUsingDirective() ||
+                context.TargetToken.IsKind(SyntaxKind.UsingKeyword) ||
```

Comment:
this would offer `using static` *anywhere* :)  that seems sad :'(

better might be to accept that at the top level a `using` is ambiguous and allow in *that* event.

Reply:
I considered keeping this tighter, but was not sure how much we care. I thought we mostly care about missing recommendations than slightly too liberal recommendations.
Here's an option: we could only add the case of a `using` keyword that belongs to a top-level statement. Would that be acceptable?

Reply:
> Here's an option: we could only add the case of a using keyword that belongs to a top-level statement.

RIght, that's what i'm saying :)